Ошибка: HTTPConnectionPool(host='30', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027ABC4970>: Failed to resolve '30' ([Errno 11001] getaddrinfo failed)"))